### DISCUSSION TITLE
BOSSEnsemble does not work with standard pip install

### DISCUSSION THREAD

**Discussion by cevheck** [alan-turing-institute/sktime#5805] [Q&A] [answered]
Hi all, when trying to fit a time series using the BossEnsemble, I keep stumbling upon the error (image provided below). I did not have custom versions of cuda yet in my environment and downloaded everything using the recommended: 
pip install sktime[all_extras] 

Can anyone help me out? Thanks in advance

![image](https://github.com/sktime/sktime/assets/61511837/95580332-fa68-401b-a0c0-0ccd949cb4a8)

**Comment by fkiraly** [comment]
Thx, could you kindly report this as a bug on the sktime issue tracker?
https://github.com/sktime/sktime/issues
important question are your package versions, e.g., numba - the instructions will tell you how to get these easily

**Reply by cevheck** [reply, marked as answer]
I found the issue, apparently float32 is required. Adding
            X_train = np.asarray(X_train).astype('float32')
solves the error
